code/game/machinery/syndicate_teleporter.dm

```diff
@@ -0,0 +1,187 @@
+		return
+	var/mob/living/carbon/human/C = owner
+	var/mob/camera/aiEye/remote/remote_eye = C.remote_control
+	var/obj/machinery/syndi_telepad/P = target
+
+	if(GLOB.cameranet.checkTurfVis(remote_eye.loc))
+		P.teleport_in(get_turf(remote_eye), owner)
+
+/obj/machinery/syndi_telepad
+	name = "hit and run telepad"
+	desc = "Used to teleport in and then quickly back out"
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "syndi-pad-idle"
+	anchored = TRUE
+	power_state = NO_POWER_USE
+	interact_offline = TRUE
+	var/cooldown = 0
+	var/cooldown_time = 3 MINUTES
+	var/retrieve_timer = 45 SECONDS //The time the teleporter teleports the person back after teleporting with it
+	var/about_to_retrieve = FALSE //If the retrieve_timer is currently running
+	var/obj/item/gps/internal/gps_signal = /obj/item/gps/internal/hit_run_teleporter
+	var/obj/machinery/computer/camera_advanced/hit_run_teleporter/linked_console
+
+/obj/machinery/syndi_telepad/Initialize(mapload)
+	. = ..()
+	try_link_console()
+	gps_signal = new gps_signal(src)
+	if(!isfloorturf(loc))
+		anchored = FALSE
+	component_parts = list()
+	component_parts += new /obj/item/circuitboard/syndi_telepad(null)
+	component_parts += new /obj/item/stack/ore/bluespace_crystal/artificial(null, 2)
+	component_parts += new /obj/item/stock_parts/matter_bin(null)
+	component_parts += new /obj/item/stock_parts/manipulator(null)
+	RefreshParts()
+
+/obj/machinery/syndi_telepad/Destroy()
+	if(linked_console)
+		linked_console.linked_pad = null
+	QDEL_NULL(gps_signal)
+	return ..()
+
+/obj/machinery/syndi_telepad/update_icon_state()
+	if(panel_open)
+		icon_state = "syndi-pad-o"
+	else
+		icon_state = "syndi-pad-idle"
+
+/obj/machinery/syndi_telepad/screwdriver_act(mob/user, obj/item/I)
+	if(default_deconstruction_screwdriver(user, "[initial(icon_state)]-o", initial(icon_state), I))
+		update_icon(UPDATE_ICON_STATE)
+		return TRUE
+
+/obj/machinery/syndi_telepad/crowbar_act(mob/living/user, obj/item/I)
+	. = TRUE
+	if(cooldown > world.time)
+		to_chat(user, "<span class='notice'>The maintenance panel is locked.</span>")
+		return
+	if(default_deconstruction_crowbar(user, I))
+		return
+
+/obj/machinery/syndi_telepad/wrench_act(mob/user, obj/item/I)
+	. = TRUE
+	if(about_to_retrieve)
+		to_chat(user, "<span class='warning'>The bolts are locked down!</span>")
+		return
+	if(default_unfasten_wrench(user, I))
+		try_link_console(TRUE)
+		return
+
+/obj/machinery/syndi_telepad/proc/try_link_console(relink)
+	if(relink && linked_console)
+		linked_console.linked_pad = null
+		linked_console = null
+	if(linked_console || !anchored)
+		return
+	for(var/obj/machinery/computer/camera_advanced/hit_run_teleporter/T in range(1, src))
+		if(T.linked_pad)
+			continue
+		linked_console = T
+		T.linked_pad = src
+		T.tele_in_action.target = src
+		return
+
+/obj/machinery/syndi_telepad/proc/teleport_out(mob/living/carbon/target)
+	if(stat & (BROKEN))
+		return
+	if(target.handcuffed && (target.buckled || target.pulledby))
```

Comment:
maybe some sort of feedback to the target would be helpful. I.e. "You feel a force trying to pull you back to your telepad... etc"